### PR TITLE
Milestone 03: Query Multiple

### PR DESCRIPTION
**Pull request by mavericks-db**
## Query Multiple

**This branch includes:**

- [x] Create a table named owners with the following columns:
   - id: integer (set it as autoincremented PRIMARY KEY)
   - full_name: string
   - age: integer
- [x] Create a table named species with the following columns:
   - id: integer (set it as autoincremented PRIMARY KEY)
   - name: string
- [x] Modify animals table:
   - Make sure that id is set as autoincremented PRIMARY KEY
   - Remove column species
   - Add column species_id which is a foreign key referencing species table
   - Add column owner_id which is a foreign key referencing the owners table
- [x] Remember all this goes in schema.sql.
- [x] Insert the following data into the owners table:
   - Sam Smith 34 years old.
   - Jennifer Orwell 19 years old.
   - Bob 45 years old.
   - Melody Pond 77 years old.
   - Dean Winchester 14 years old.
   - Jodie Whittaker 38 years old.
- [x] Insert the following data into the species table:
   - Pokemon
   - Digimon
- [x] Modify your inserted animals so it includes the species_id value:
   - If the name ends in "mon" it will be Digimon
   - All other animals are Pokemon
- [x] Modify your inserted animals to include owner information (owner_id):
   - Sam Smith owns Agumon.
   - Jennifer Orwell owns Gabumon and Pikachu.
   - Bob owns Devimon and Plantmon.
   - Melody Pond owns Charmander, Squirtle, and Blossom.
   - Dean Winchester owns Angemon and Boarmon.
- [x] Remember these insertions and modifications happen in data.sql.
- [x] Write queries (using JOIN) to answer the following questions:
   - What animals belong to Melody Pond?
   - List of all animals that are pokemon (their type is Pokemon).
   - List all owners and their animals, remember to include those that don't own any animal.
   - How many animals are there per species?
   - List all Digimon owned by Jennifer Orwell.
   - List all animals owned by Dean Winchester that haven't tried to escape.
   - Who owns the most animals?
- [x] Take a screenshot of the results of your queries.
- [x] Remember to add the screenshot with the results of your queries to your Pull Request description.

full_screenshot

### CODE REVIEW

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 1]
```diff
@@ -11,3 +11,32 @@ CREATE TABLE animals (
...
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name TEXT
+);
+
+ALTER TABLE animals
+DROP COLUMN id;
+
```
[the remark this answers is not in the document]
Comment (mavericks-db) [reply schema.sql:28]: Revised on commit id [49120a4](https://github.com/mavericks-db/vet-clinic-database/pull/3/commits/49120a4d31a0d1d0802d84caa58788a5cdb66d09)

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @mavericks-db, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The PR title and description looks professional 👍
- Descriptive commit messages 👍
- You did well to make the changes requested by the previous reviewer 👍
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mavericks-db** [comment]
Thank you 😊

**Pull request merged**